src/handle_join_leave.cxx

```diff
@@ -596,6 +596,9 @@ void raft_server::handle_join_leave_rpc_err(msg_type t_msg, ptr<peer> p) {
               p->get_id() );
         config_changing_ = false;
         reset_srv_to_join();
+
+        cb_func::Param param(id_, leader_, p->get_id());
+        invoke_callback(cb_func::ServerJoinFailed, &param);
```

Comment:
There is one more place to call it:
https://github.com/eBay/NuRaft/blob/99eeef34a2620686e0dd40ad7fbd5cab561140fc/src/handle_join_leave.cxx#L96-L98
Not only by explicit error (this function), but also by timeout.

Reply:
Done, thanks